docs/specs/clients/notify/client-sdk-api.md

```diff
@@ -56,17 +58,33 @@ abstract class Client {
     encryptedMessage: string,
   }): Promise<NotifyMessage>;
   
+  // Generates and returns SIWE message along with
+  // Cacao Payload with identity key
+  public abstract prepareRegistration(params: {
+    account: string;
+    domain: string,
+    // Defaulted to false if not passed
+    allApps?: boolean,
+  }): {
+    registerParams: NotifyRegistrationParams,
+    message: string
+  };
+  
   // "Logs in" this client to notifications for the specified account. This involves:
-  // - Generating, signing, saving to local stroage, and registering an Identity Key, if one didn't already exist in local storage
+  // - Verifies signature by reconstructing formatted message using provided payload
+  // - Expected payload comes from prepareRegistration
+  // - Once verified, registers the identity key on the keyserver
   // - Calls `wc_notifyWatchSubscriptions`, watches for future subscription updates, and subscribes to all notification topics
   // Returns the Identity Key public key. Method should throw 'signatureRejected' if any errors comes from onSign promise. 
```

Comment:
```suggestion
  // Returns the Identity Key public key. Method should throw 'signatureRejected' if any errors comes from onSign promise. 
```

Should this and the response value be removed now?

Reply:
`signatureRejected` definitely, but we can still return identity key as a form of success

Reply:
> signatureRejected definitely

No because there is no onSign promise anymore

> we can still return identity key as a form of success

Seems useless, but ok

Reply:
yes, sorry I was unclear `definitely` as in definitely should be removed, already committed removing it